platforms/common/uORB/uORBDeviceMaster.cpp

```diff
@@ -392,18 +392,25 @@ void uORB::DeviceMaster::showTop(char **topic_filter, int num_filters)
 			PX4_INFO_RAW(CLEAR_LINE "%-*s INST #SUB RATE #Q SIZE\n", (int)max_topic_name_length - 2, "TOPIC NAME");
 			cur_node = first_node;
 
+			size_t total_size = 0;
+
 			while (cur_node) {
 
 				if (!print_active_only || (cur_node->pub_msg_delta > 0 && cur_node->node->subscriber_count() > 0)) {
 					PX4_INFO_RAW(CLEAR_LINE "%-*s %2i %4i %4i %2i %4i \n", (int)max_topic_name_length,
 						     cur_node->node->get_meta()->o_name, (int)cur_node->node->get_instance(),
 						     (int)cur_node->node->subscriber_count(), cur_node->pub_msg_delta,
 						     cur_node->node->get_queue_size(), cur_node->node->get_meta()->o_size);
+
+
+					total_size += cur_node->pub_msg_delta * cur_node->node->get_meta()->o_size;
 				}
 
 				cur_node = cur_node->next;
 			}
 
+			PX4_INFO_RAW("\nTotal publication rate: %.1f kB/s\n", (double)total_size / 1000.);
```

Comment:
Not that it really matters here, but I'd avoid the div in double.
```suggestion
			PX4_INFO_RAW("\nTotal publication rate: %.1f kB/s\n", (double)(total_size / 1000.f));
```